docs/admin.rst

```diff
@@ -1,9 +1,71 @@
 Administrator Guide
 =====================
 
-At this point, you (the administrator) should have your own username and
-password, plus two-factor authentication through either the Google
-Authenticator app on your smartphone or a YubiKey.
+You (the administrator) should have your own username and password, plus
+two-factor authentication through either the Google Authenticator app on
+your smartphone or a YubiKey.
+
+.. _Responsibilities:
+
+Responsibilities
+----------------
+
+The SecureDrop architecture contains multiple hardened servers, and while we have
+automated many of the installation and maintenance tasks, a skilled Linux
+administrator and some manual intervention is required to responsibly run the system.
+
+This section outlines the tasks the administrator is responsible for in order to
+ensure that the SecureDrop server continues to be a safe place for sources to
+talk to journalists.
+
+Keep your SecureDrop Server Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should maintain awareness of SecureDrop updates and take any required
+manual action if requested in the `SecureDrop Release Blog`_. We recommend
+subscribing to the `SecureDrop RSS Feed`_ to stay apprised of new updates.
+
+Most often, the SecureDrop server will automatically update via apt. However,
+occasionally you will need to run the Ansible playbooks. We will inform you in
+the release blog when this is the case. If you are onboarded to our support portal,
```

Comment:
Agreed, linking to support portal is appropriate here, as we do it elsewhere in the docs already. Prime candidate for a reusable include as described in #1319, but to keep the wheels turning, let's just copy/paste it: `_SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/`